src/cpp/include/openvino/genai/perf_metrics.hpp

```diff
@@ -150,5 +150,7 @@ struct OPENVINO_GENAI_EXPORTS PerfMetrics {
     RawPerfMetrics raw_metrics;
 };
 
+MeanStdPair calc_mean_and_std(const std::vector<MicroSeconds>& durations);
```

Comment:
are external users are supposed to use this method? if no, let's hide in the `src` folder

Reply:
Removed from public api